lib/static/modules/utils.js

```diff
@@ -112,19 +112,58 @@ function setStatusForBranch(nodes, suitePath) {
     setStatusForBranch(nodes, suitePath.slice(0, -1));
 }
 
-function shouldSuiteBeShown(suite, filterByName) {
+function shouldSuiteBeShownByName(suite, filterByName) {
     const suiteFullPath = suite.suitePath.join(' ');
 
     if (suiteFullPath.includes(filterByName)) {
         return true;
     }
     if (suite.hasOwnProperty('children')) {
-        return suite.children.some(child => shouldSuiteBeShown(child, filterByName));
+        return suite.children.some(child => shouldSuiteBeShownByName(child, filterByName));
     }
 
     return false;
 }
 
+function shouldSuiteBeShownByBrowser(suite, filteredBrowsers) {
+    if (suite.hasOwnProperty('browsers')) {
```

Comment:
It's strange that here `browsers` is processed first, but in the next function - `children`. Same order is better, I suppose